src/data-pipeline/spark-etl/src/main/java/software/aws/solution/clickstream/TransformerV2.java

```diff
@@ -527,7 +527,7 @@ private Optional<Dataset<Row>> extractUser(final Dataset<Row> dataset) {
 
         Dataset<Row> profileSetDataset = userDataset
                 .filter(col(EVENT_NAME)
-                        .isin("user_profile_set", "_user_profile_set", "_profile_set"));
+                        .isin("user_profile_set", "_user_profile_set", "_profile_set", "_first_open"));
```

Comment:
Is there an existing test case for this?

Reply:
added Unit test